src/types/json.h

```diff
@@ -217,6 +217,32 @@ struct JsonValue {
     return results;
   }
 
+  StatusOr<std::vector<size_t>> GetBytes(std::string_view path, JsonStorageFormat format,
+                                         int max_nesting_depth = std::numeric_limits<int>::max()) const {
+    std::vector<size_t> results;
+    Status s;
+    try {
+      jsoncons::jsonpath::json_query(value, path, [&](const std::string & /*path*/, const jsoncons::json &origin) {
+        std::string buffer;
+        JsonValue query_value(origin);
+        if (format == JsonStorageFormat::JSON) {
+          s = query_value.Dump(&buffer, max_nesting_depth);
+        } else if (format == JsonStorageFormat::CBOR) {
+          s = query_value.DumpCBOR(&buffer, max_nesting_depth);
+        }
+        results.emplace_back(buffer.size());
+      });
+    } catch (const jsoncons::jsonpath::jsonpath_error &e) {
+      return {Status::NotOK, e.what()};
+    }
+    if (!s) return {Status::NotOK, s.Msg()};
+    if (results.size() == 0) {
+      results.emplace_back(0);
+    }
```

Comment:
It's wrong.

Please use redis-stack and redis-cli to confirm the correct result for this case.